### PR TITLE
fix images urls and base uri

### PR DESCRIPTION
**Pull request by m-kharbat**
@chaudum please review

http://g.recordit.co/07Puo8e8em.gif

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,4 @@ eggs/
 .idea/
 .tmp/
 dist/
+app/tests/coverage/
```
Comment (chaudum) [inline .gitignore]: this is not related. please make a separate commit

File: CHANGES.txt [hunk 2]
```diff
@@ -5,6 +5,8 @@ Changes for Crate Admin Interface
 Unreleased
 ==========
 
+ - Fix: Corrected the image urls and base_uri that were broken in the build
+
  - Fix: do not try to show load history when load information from node is
```
Comment (chaudum) [inline CHANGES.txt]: reflect release 1.0.0 in changes is not on master? - needs to be done before!

File: CHANGES.txt [hunk 3]
```diff
@@ -5,6 +5,8 @@ Changes for Crate Admin Interface
 Unreleased
 ==========
 
+ - Fix: Corrected the image urls and base_uri that were broken in the build
```
Comment (mikethebeer) [inline CHANGES.txt]: Suggestion: Fixed an issue that caused incorrect URL paths if the project gets built.
